opennebula/resource_opennebula_security_group.go

```diff
@@ -257,13 +258,42 @@ func resourceOpennebulaSecurityGroupRead(d *schema.ResourceData, meta interface{
 	d.Set("gname", securitygroup.GName)
 	d.Set("permissions", permissionsUnixString(*securitygroup.Permissions))
 
-	description, _ := securitygroup.Template.GetStr("DESCRITPION")
+	description, _ := securitygroup.Template.Get(sgk.Description)
 	d.Set("description", description)
 
 	if err := d.Set("rule", generateSecurityGroupMapFromStructs(securitygroup.Template.GetRules())); err != nil {
 		log.Printf("[WARN] Error setting rule for Security Group %x, error: %s", securitygroup.ID, err)
 	}
 
+	err = flattenSecurityGroupTags(d, &securitygroup.Template)
```

Comment:
DESCRIPTION wiil be in tags as it's not deleted

Reply:
Actually not.
Only declared tags in the TF file are searched in the flatten function